packages/genesys-spark-components/package.json

```diff
@@ -41,8 +41,8 @@
     "svgo": "find ./src/components/stable/gux-icon/icons/**/*.svg -type f ! -path '*/legacy/*' -exec svgo {} +",
     "test": "TZ=UTC stencil test --spec --e2e",
     "test.ci": "TZ=UTC npm run test.ci.spec",
-    "test.ci.e2e": "TZ=UTC stencil test --e2e --maxWorkers=1",
-    "test.ci.spec": "TZ=UTC stencil test --spec --maxWorkers=1",
+    "test.ci.e2e": "TZ=UTC stencil test --e2e --maxWorkers=1 --workerIdleMemoryLimit=500MB",
+    "test.ci.spec": "TZ=UTC stencil test --spec --maxWorkers=1 --workerIdleMemoryLimit=500MB",
```

Comment:
https://jestjs.io/docs/configuration/#workeridlememorylimit-numberstring

We have to use version 20.9.0 of node as that is all that is available in Jenkins. There is a bug in this version of node that was fixed in 20.10.0 :(. This is a workaround created by jest for situations where devs are stuck on a version of node that has the bug.

Reply:
Once this is merged I will create a Jira issue to remove it once we move to a newer version of node.